source/endpoints/devices/Activator.py

```diff
@@ -26,18 +26,14 @@ class Activator(Resource):
     @namespace.doc("get_activator")
     @namespace.marshal_with(activator)
     def get(self, device_id, activator_id):
-        """ Fetch a given action of a device """
-        device = device_database.get_device(device_id)
-        return device.get_activator(activator_id)
+        """ Fetch a given activator of a device """
+        return business_logic_activator.get_activator_from_database_by_device_and_activator_id(device_id, activator_id)
```

Comment:
Why not `.get_activator_from_database`? Why the need to suddenly encode arguments in the name?